perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -310,10 +310,11 @@ sub confirm_subscription : Private {
     if ($already_confirmed) {
         my $payment = $p->get_extra_field_value('payment');
         $payment = sprintf( '%.2f', $payment / 100 );
-        $p->add_to_comments({
+        my $comment = $p->add_to_comments({
             text => "Payment confirmed, reference $reference, amount £$payment",
             user => $p->user,
         });
+        $p->cancel_update_alert($comment->id);
```

Comment:
I've left this in, but can't provoke the tests to send an update whether it's here or not - as mentioned in other comment, I was pretty sure I was able to get the tests to pass or fail dependent on this being here, but am a bit stumped.